azure-local/update/update-best-practices.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Best Practices for managing Azure Local Update Management
+description: Learn the best practices for managing Azure Local environments.
+author: alkohli
+ms.topic: overview
+ms.date: 09/02/2025
+ms.author: alkohli
+ms.reviewer: alkohli
+ms.service: azure-local
+---
+
+# Best practices for managing Azure Local environments
+
+This article provides an overview of Azure Local update management, including best practices and common pitfalls to help keep Azure Local secure, up to date, and compliant. It is intended for IT decision-makers, infrastructure architects, and operations teams responsible for Azure Local deployments.
+
+## Understand Azure Local update management
+
+Azure Local supports update management through Azure Update Manager (AUM) and Azure Resource Manager (ARM) templates.
```

Comment:
@arduppal: Here we are stating both AUM and ARM templates for update management. Do we need to revise this section also?

Reply:
yes remove ARm temeplate